src/PaymentGateways/Gateways/PayPalStandard/Webhooks/WebhookValidator.php

```diff
@@ -58,7 +59,10 @@ public function verifyEventSignature(array $eventData)
         if ('VERIFIED' !== $apiResponse['body']) {
             Log::error(
```

Comment:
Let's change this to a warning.

Reply:
fixed: https://github.com/impress-org/givewp/compare/2057d9cfadd8...cacdc412417e